### PR TITLE
Refactoring code

### PR DESCRIPTION
**Pull request by toby3d**
- Moved all docs to separate folder
- Renamed '3rd-Party Licenses' on 'third_party'
- Moved some tests to 'test' directory
- Merge values and types blocks
- Renamed 'types.go' to 'models.go'
- Some little fixes

### CODE REVIEW

File: test/gramework_test.go [hunk 1]
```diff
@@ -64,21 +65,15 @@ func TestGrameworkHTTP(t *testing.T) {
 	app.SPAIndex("./nanotime.s")
 	app.GET("/sdnc_static/dist/*static", app.ServeDirNoCache("./"))
 	app.GET("/sdncc_static/dist/*static", app.ServeDirNoCacheCustom("./", 0, false, false, []string{}))
-	app.MethodNotAllowed(func(ctx *Context) {
+	app.MethodNotAllowed(func(ctx *gramework.Context) {
 		ctx.WriteString("GTFO")
 	})
 	app.UsePre(func() {
 		preCalled = true
 	})
```
Comment (kirillDanshin) [inline test/gramework_test.go]: Error return value is not checked

Comment (toby3d) [reply test/gramework_test.go]: I need use this:
```go
if err != nil {
	t.Error(err.Error())
}
```
or something else?

Comment (kirillDanshin) [reply test/gramework_test.go]: yes, but also add a log warn so we can understand what's going on

File: apiClient/models.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package apiClient
```
Comment (kirillDanshin) [inline apiClient/models.go]: please name the file types.go instead of models. we don't have any models in this project.

**Comment by toby3d** [comment]
Should I rename `apiClient` to just `client`?

**Comment by kirillDanshin** [comment]
upd: after consulting with gramework's lawyers, please rename third_party to third_party_licenses

**Comment by toby3d** [comment]
@kirillDanshin 👌

File: test/grypto_test.go [hunk 3]
```diff
@@ -1,17 +1,22 @@
-package grypto
+package test
 
 import (
 	"testing"
 
 	"github.com/gramework/utils/grand"
+	"github.com/toby3d/gramework/grypto"
```
Comment (kirillDanshin) [inline test/grypto_test.go]: w~tf~hy?

**Comment by kirillDanshin** [comment]
please rebase this branch

**Pull request merged**